docs/background.rst

```diff
@@ -9,54 +9,67 @@ For assembling the widely used CoRE-MOF database (see `first <https://pubs.acs.o
+ * the structure graph hash (:py:attr:`~mofchecker.MOFChecker.graph_hash`) is like the scaffold hash, but also considers the atomic elements (nodes) in the graph
 
+What is it useful for?
+......................
 
-What can it be used for?
-............................
+The scaffold and structure graph hashes allow to quickly identify duplicates in large structure databases.
 
-The most important use case for the hashes is to identify duplicates in databases for which it is unfeasible to perform :math:`N^2` graph isomorphirsm checks.
-Note that the definition of duplicate, under the :py:attr:`~mofchecker.MOFChecker.graph_hash`, is similar to the one proposed by `Barthel et al. <https://pubs.acs.org/doi/pdf/10.1021/acs.cgd.7b01663>`_ :
+The concept of duplicate structures, as defined by comparing their :py:attr:`~mofchecker.MOFChecker.graph_hash`, closely follows the one proposed by `Barthel et al. <https://pubs.acs.org/doi/pdf/10.1021/acs.cgd.7b01663>`_ :
 
     However, from a MOF point of view two structures are considered identical if they share the same bond network, with respect to the atom types and their embedding:
     i.e., if two structures can in principle be deformed into each other without breaking and forming bonds.
 
-The scaffold hash can be useful to find families of related MOFs. For example, all members of the unfunctionalized MOF-74 familiy would group under the same hash. Similarly, all functionalized UiO-66 structures would group under the same hash as long as the position and the size (e.g., one atom) of the functionalization is the same.
-
-What can go wrong?
-.......................
-
-There are multiple ways in which the hash can yield results different from what one would expect.
-
-1. Incorrect bonding network
-2. Did not reduce to primitive cell
-3. Unlucky hash clash (Weisfeiler Lehman has some `edge cases <https://informaconnect.com/beyond-weisfeiler-lehman-using-substructures-for-provably-expressive-graph-neural-networks/>`_)
+The scaffold hash can be useful to find families of related MOFs.
+For example, all members of the (unfunctionalized) MOF-74 family, such as Ni-MOF-74 or Mg-MOF-74, group under the same hash.
 
+.. image:: _static/hash_comparison_mof_74.jpg
+  :width: 600
+  :alt: Ni-MOF-74 and Mg-MOF-74 have the same scaffold hash, but different structure graph hashes.
 
 How does it work?
 ....................
 
-Under the hood, mofchecker uses `pymatgen <http://pymatgen.org/>`_ to analyze the bonding network to create a structure graph.
-We then apply the Weisfeiler Lehman algorithm (as implemented in `networkx <https://networkx.org/>`_) to compute the hash.
+mofchecker
+
+ #. reduces the structure to the primitive cell using `pymatgen <http://pymatgen.org/>`_ and `spglib <https://spglib.github.io/spglib/>`_ (use ``primitive=False`` to disable this)
```

Comment:
not important for this PR but we should address #23 since we should not ask users to use a private method.